images/linux/scripts/installers/kubernetes-tools.sh

```diff
@@ -32,7 +37,20 @@ if ! command -v helm; then
     exit 1
 fi
 
+# Run tests to determine that the software installed as expected
+echo "Testing to make sure that minikube was installed"
+if ! command -v minikube; then
+    echo "minikube was not installed"
+    exit 1
+fi
+
+# Cache Kubernetes images (images, tarballs) into .minikube folder.
+echo "Caching Kubernetes images"
```

Comment:
@medyagh Any idea how big these are? Might make sense to leave them off the cache for now.

Reply:
@alepauly , Based on issue description, ~500 MB.
I think we should leave cache off for now because we have only about 1.5GB of free disk space on Ubuntu. That is pretty small.

Reply:
agreed. I understand the benefit of preloading but we can't afford the space at this time. @medyagh can you update please? thanks!